client/src/modules/profiles/pages/userProfile.tsx

```diff
@@ -21,7 +21,7 @@ export const UserProfilePage = () => {
   const logout = useLogout();
   const router = useRouter();
 
-  const confirmDelete = useConfirmDelete({ doubleConfirm: true });
+  const confirmDelete = useConfirmDelete({ buttonText: 'Yes, im sure' });
```

Comment:
I don't know what happens but after deleting the using, 

auth with google is deleted

![image](https://user-images.githubusercontent.com/88248797/195658698-f897e00c-2a64-4d2b-a7e5-40f74019678a.png)

---

I have no issue with it, just wondering if we okay with single confirm?

Why did we have double confirm before?

Reply:
Is that instance owner logged in?

Reply:
nope I deleted the instance owner 🤣

Reply:
Exactly :slightly_smiling_face: